app/web/src/newhotness/ApplyChangeSetModal.vue

```diff
@@ -80,11 +81,26 @@
                 ),
               )
             "
-            :loadingText="approvalsEnabled ? 'undefined' : 'Applying Changes'"
-            :loading="approvalsEnabled ? false : applyInFlight"
-            :disabled="approvalsEnabled ? false : disallowApply"
             pill="Cmd + Enter"
-            @click="debouncedApplyOrRequestApproval"
+            @click="debouncedRequestApproval"
+          />
+          <VButton
```

Comment:
Split into two buttons to reduce confusion. Yes, there's more cleanup and refactor work needing to be done in this domain, but it's piece-by-piece work in progress.